frontend/app/src/store/notifications/index.ts

```diff
@@ -118,11 +118,11 @@ export const useNotifications = defineStore('notifications', () => {
     try {
       const object = JSON.parse(message);
       if (!object.type) {
-        notify(handleLegacyMessage(message, isWarning));
+        notify(handleLegacyMessage(message, isWarning, tc));
```

Comment:
This way it doesn't log an error. @lukicenturi if you have any other ideas we can try them